.rubocop.yml

```diff
@@ -0,0 +1,508 @@
+require: rubocop-rspec
```

Comment:
Ah, I was not aware of that one (and I like it more [on the surface]). The version here is copied from one of our modules, and is only meant as a starting point. we're currently in no position to activate rubocop testing on our modules anyways. I'll make a note on the appropriate ticket. Thanks for your scrutiny :-)